app/services/register_and_partner_api/sync_users.rb

```diff
@@ -8,10 +8,10 @@ class SyncUsers
     }.freeze
 
     CIP_TYPES = {
-      ambition: "Ambition Institute",
-      edt: "Education Development Trust",
-      teach_first: "Teach First",
-      ucl: "UCL",
+      ambition: "ambition",
+      edt: "edt",
+      teach_first: "teach_first",
+      ucl: "ucl",
```

Comment:
what is the point in this object?

Reply:
Its main goal is to avoid sending names of core induction programme between R&P and E&L - cause those names are not the same 😢 

So instead of sending names we send an enum. And this is one way you can have an enum in Ruby code. There is a carbon copy of this object in R&P.